scylla/src/utils/mod.rs

```diff
@@ -1,3 +1,94 @@
 pub(crate) mod parse;
 
 pub mod test_utils;
+
+use std::error::Error;
+
+use tracing::{error, Span};
+
+/// Constructs an OpenTelemetry span at the trace level.
+///
+/// Basically does the same as [tracing::trace_span!], only adding two (empty) fields:
+/// `otel.status_code` and `otel.status_message`.
+#[macro_export]
+macro_rules! otel_span {
```

Comment:
Let's not make the OTel-related traits public. Traits that just provide helper methods for our - but not user's - convenience shouldn't be accessible from outside.